userguide/jails.rst

```diff
@@ -817,44 +772,45 @@ available for a jail.
-   |              | deleting a :guilabel:`Mount Point`. See                       |
+   |              | deleting a :guilabel:`MOUNT POINT`. See                       |
    |              | :ref:`Additional Storage` for more details.                   |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Restart      | Stop and immediately start an :literal:`up` jail.             |
+   | RESTART      | Stop and immediately start an :literal:`up` jail.             |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Start        | Start a jail that has a current :guilabel:`Status` of         |
+   | START        | Start a jail that has a current :guilabel:`STATE` of          |
    |              | *down*.                                                       |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Stop         | Stop a jail that has a current :guilabel:`Status` of          |
+   | STOP         | Stop a jail that has a current :guilabel:`STATE` of           |
    |              | *up*.                                                         |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Update       | Runs `freebsd-update                                          |
+   | UPDATE       | Runs `freebsd-update                                          |
    |              | <https://www.freebsd.org/cgi/man.cgi?query=freebsd-update>`__ |
    |              | to update the jail to the lateset patch level of the          |
    |              | installed FreeBSD release.                                    |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Shell        | Access a *root* command prompt to interact with a jail        |
+   | SHELL        | Access a *root* command prompt to interact with a jail        |
    |              | directly from the command line. Type :command:`exit` to       |
    |              | leave the command prompt.                                     |
    |              |                                                               |
    +--------------+---------------------------------------------------------------+
-   | Delete       | Delete the jail, all of the jail's contents, and all          |
+   | DELETE       | Delete the jail, all of the jail's contents, and all          |
    |              | associated :ref:`Snapshots`. Back up the jail's data,         |
```

Comment:
Rephrase so the first word is not a verb that can be read as imperative:
`` Caution: deleting the jail also deletes all of the jail contents and all associated :ref:`snapshots <Snapshots>`. Back up the jail data... ``